### PR TITLE
StackCtrl added to the list.

### PR DESCRIPTION
**Pull request by ismail-yilmaz**
Adds a simple container ctrl to the UppHub that displays one child ctrl at a time. (similar to GTK::Stack or QT::QStackWidget)

Please check.

### CODE REVIEW

File: nests.json [hunk 1]
```diff
@@ -416,6 +416,17 @@
...
+    },
+    {
+      "name": "StackCtrl",
+      "packages": [
+        "StackCtrl"
+      ],
+      "description": "A simple widget stacking container for U++ that displays only one ctrl at a time.",
+      "repository": "https://github.com/ismail-yilmaz/StackCtrl.git",
+      "status": "stable",
+      "category": "CtrlLib",
```
Comment (klugier) [inline nests.json:428]: Use widget category instead.

File: nests.json [hunk 2]
```diff
@@ -416,6 +416,17 @@
...
-    }    
+    },
+    {
+      "name": "StackCtrl",
+      "packages": [
+        "StackCtrl"
+      ],
+      "description": "A simple widget stacking container for U++ that displays only one ctrl at a time.",
+      "repository": "https://github.com/ismail-yilmaz/StackCtrl.git",
+      "status": "stable",
```
Comment (klugier) [inline nests.json:427]: Shouldn't be it experimental at least for some time? This is new widget that was mainly used by you.

Comment (ismail-yilmaz) [reply nests.json]: OK. changed the category and status.

File: nests.json (hunk 1 above)
Comment (ismail-yilmaz) [reply nests.json]: Done

**Review by klugier — APPROVED** [review]
LGTM!

**Pull request merged**